src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -124,9 +124,7 @@ private HashSet<string> GetSourceMemberNames()
 
     private Dictionary<string, IMappableMember> GetTargetMembers()
     {
-        return BuilderContext
-            .SymbolAccessor.GetAllAccessibleMappableMembers(Mapping.TargetType)
-            .ToDictionary(x => x.Name, StringComparer.OrdinalIgnoreCase);
+        return BuilderContext.SymbolAccessor.GetAllAccessibleMappableMembers(Mapping.TargetType).ToDictionary(x => x.Name);
```

Comment:
Doesn't
`src\Riok.Mapperly\Descriptors\MappingBodyBuilders\BuilderContext\NewInstanceBuilderContext.cs:10` and `src\Riok.Mapperly\Descriptors\MappingBodyBuilders\BuilderContext\NewInstanceContainerBuilderContext.cs:17` have the same problems for configurations?